rules/code-quality/tests/Rector/Expression/InlineIfToExplicitIfRector/Fixture/fixture.php.inc

```diff
@@ -27,6 +28,9 @@ class Fixture
         if (is_null($userId)) {
             $userId = 5;
         }
+        if (!is_null($userId)) {
+            $userId = 5;
+        }
     }
```

Comment:
This could be quite hard to debug, as there are 2 uses cases in one file now.

Could you split it into 2 fixture files?

Reply:
implemented.